visualdl/server/visualDL

```diff
@@ -298,11 +298,18 @@ def individual_audio():
 
 @app.route('/data/plugin/graphs/graph')
 def graph():
-    # TODO(ChunweiYan) need to add a config for whether have graph.
-    if graph_image_path is None or not os.path.isfile(graph_image_path):
-        data = {'url': ''}
+    # TODO(daming-lu): rename variables because we switched from static graph generated by graphviz
+    # to d3/dagre drawn svg
+    if graph_image_path is not None:
```

Comment:
Will we generate static image? We might be able to remove block

Reply:
Like the TODO mentioned, we need to rename and remove some stuff because we were using GraphViz previously. I will do it after the release